src/commands/protocols/update/receiver/delete-pending-state-command.js

```diff
@@ -111,6 +113,7 @@ class DeletePendingStateCommand extends Command {
                 return true;
             }
         }
+        this.logger.debug(`No pending state exists for assertion id: ${assertionId}`);
```

Comment:
Add operation id

Reply:
operationId does not exist in this method